swap/src/network/request_response.rs

```diff
@@ -21,21 +21,34 @@ const BUF_SIZE: usize = 1024 * 1024;
 /// Messages Bob sends to Alice.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum BobToAlice {
-    SwapRequest(bob::SwapRequest),
+    SwapRequest(Box<bob::SwapRequest>),
```

Comment:
❓ Why is there a `Box` needed for the `SwapRequest` now?

Reply:
a910bc204636cf5c85f520168aab3c5061ba31d7